packages/test-sync/src/androidTest/kotlin/io/realm/kotlin/test/mongodb/shared/FlexibleSyncConfigurationTests.kt

```diff
@@ -155,6 +156,18 @@ class FlexibleSyncConfigurationTests {
         assertTrue(config.initialSubscriptions!!.rerunOnOpen)
     }
 
+    @Test
+    fun durability() {
```

Comment:
Also missing a general `SyncConfiguration` test for partition-based sync. Could be placed in `SyncConfigTests`